themes/bootstrap5/templates/Auth/MultiILS/login.phtml

```diff
@@ -0,0 +1,48 @@
+<?php $account = $this->auth()->getManager(); ?>
+<?php $sessionInitiator = $account->getSessionInitiator($this->serverUrl($this->url('myresearch-home'))); ?>
+<?php if (!$sessionInitiator): // display default login form if no login URL provided ?>
+  <?php
+    $currentTarget = $this->request->get('target');
+    $loginTargets = $this->auth()->getManager()->getLoginTargets();
+    if (!$currentTarget || !in_array($currentTarget, $loginTargets)) {
+      $currentTarget = $this->auth()->getManager()->getDefaultLoginTarget();
+    }
+  ?>
+  <form method="post" action="<?=$this->url('myresearch-home')?>" name="loginForm" class="form-login">
+    <?=$this->auth()->getLoginFields()?>
+    <?php if ($account->supportsPersistentLogin()) : ?>
+      <div class="form-group">
+        <label>
+          <input type="checkbox" name="remember_me"<?=$this->request->get('remember_me') ? ' checked' : ''?>>
+          <?=$this->transEsc('remember_me', ['%%days%%' => $account->getPersistentLoginLifetime()])?>
+        </label>
+      </div>
+    <?php endif; ?>
+    <input type="hidden" name="auth_method" value="<?=$this->escapeHtmlAttr($account->getAuthMethod())?>">
+    <input type="hidden" name="csrf" value="<?=$this->escapeHtmlAttr($account->getCsrfHash())?>">
+    <div class="form-group">
+      <input class="btn btn-primary" type="submit" name="processLogin" value="<?=$this->transEscAttr('Login')?>">
+      <?php if ($account->supportsCreation()): ?>
+        <a class="btn btn-link createAccountLink" href="<?=$this->url('myresearch-account') ?>?auth_method=<?=$account->getAuthMethod()?>"><?=$this->transEsc('Create New Account')?></a>
+      <?php endif; ?>
+      <a id="recovery_link" class="btn btn-link recover-account-link hidden" href="#"><?=$this->transEsc('Forgot Password')?></a>
+    </div>
+  </form>
+<?php else: ?>
+  <a href="<?=$this->escapeHtmlAttr($sessionInitiator)?>" class="btn btn-link" data-lightbox-ignore><?=$this->transEsc('Institutional Login')?></a>
+<?php endif; ?>
+
+<?php
+    $recovery = [];
+    foreach ($account->getLoginTargets() as $target) {
+        $recovery[$target] = $account->supportsRecovery(target: $target)
+          ? $this->url('myresearch-recover', [], ['query' => ['auth_method' => $account->getAuthMethod(), 'target' => $target]])
+          : false;
+    }
+
+    $recovery = json_encode($recovery);
+    $script = "displayILSPasswordRecoveryLink($recovery, 'login_{$topClass}_');";
+
+    // Inline the script for lightbox compatibility
+    echo $this->inlineScript(\Laminas\View\Helper\HeadScript::SCRIPT, $script, 'SET');
```

Comment:
This needs to use the assetManager once the latest dev is merged in.

Reply:
Merged and changed.